translations/consolinno-energy.en.ts

```diff
@@ -2444,12 +2444,12 @@ Use a timeout value greater or equal to 10 ms.</source>
     </message>
     <message>
         <source>CHANGENOTIFICATION_PLACEHOLDER</source>
-        <translation type="unfinished">&lt;h3&gt;The app has been updated.&lt;/h3&gt;
-            &lt;p&gt;Version 1.7.0 (November 30, 2024)&lt;/p&gt;
-            &lt;h4&gt;New features&lt;/h4&gt;
+        <translation>&lt;h3&gt;The app has been updated.&lt;/h3&gt;
+            &lt;p&gt;Version 1.7.1 (December 11, 2024)&lt;/p&gt;
+            &lt;h4&gt;Improvements&lt;/h4&gt;
```

Comment:
```suggestion
            &lt;p&gt;Version 1.8.0 (2025)&lt;/p&gt;
            &lt;h4&gt;New features&lt;/h4&gt;
```
hier sollte man auch schon anpassen :)